lib/index.ts

```diff
@@ -64,8 +93,8 @@ function getPine(param: {
 			options: {
 				method: string;
 				url: string;
-				body?: PinejsClientCoreFactory.AnyObject;
-			} & PinejsClientCoreFactory.AnyObject,
+				body?: AnyObject;
+			} & AnyObject,
 		) {
 			return auth.hasKey().then(function(hasKey) {
```

Comment:
This can be converted to async/await now that pinejs-client-core accepts native promises